### PR TITLE
Added CODEOWNERS file

### PR DESCRIPTION
**Pull request by macterra**
https://help.github.com/articles/about-codeowners/

### CODE REVIEW

File: .github/CODEOWNERS
Comment (macterra) [inline .github/CODEOWNERS:5]: I added all 4 of us with the intent of configuring the PR to require 2 approvals (author is implicit). That way in the normal use case we are all automatically added to each PR, Kiarash can opt in or out as he likes and the remainder must approve but Kiarash can also step in if one of us is unable to review for whatever reason.

image

**Review by ealefad — APPROVED** [review]

**Review by KiarashN — APPROVED** [review]

**Review by r-soltani — APPROVED** [review]
It would be great if we could script the minimum number of required approvals. I couldn't find anything on that either.

**Pull request merged**
